vpc/service/assign_addresses_v3.go

```diff
@@ -194,7 +169,7 @@ WHERE assignment_id = $1
 	if !completed {
 		if deleteUnfinishedAssignment {
 			// Delete the assignment
-			_, err = tx.ExecContext(ctx, "DELETE FROM assignments WHERE assignment_id = $1", assignmentID)
+			_, err = tx.ExecContext(ctx, "DELETE FROM assignments WHERE assignment_id = $1", taskID)
```

Comment:
Is this right here? Is assignment_id really a taskId in this context?

Reply:
There are two "ids" in an assignment table:
* id: The PK, which is auto-generated.
* assignment_id: Use the task ID passed by caller (titus-vpc-tool/titus-agent) in the API calls such as `AssignIPV3`.

In the code, some places we have "assignment.id" (referring to `id`) and some places we have `assignment_id`, which are kind of confusing. I'd like to refer to the second using `taskId` to avoid confusing.

Eventually, this query will also be moved to its own function like `DeleteAssignmentByTaskID`. Because the caller knows nothing about the two IDs, what I want to achieve is that the service code also knows nothing about it. Eventually, only DB-related code should know that when deleting an assignment for a given task ID, it should use the task ID as the assignment ID in the DB query.

Reply:
To answer the question, yes in this context, `assignment_id` is really the task ID. If you trace the callers, you'll find they all pass `req.TaskId` as the `assignment_id`. I'll make sure things are clearer after some other refactors to separate code and data.